frontend/projects/abtesting/src/app/core/experiments/store/experiments.reducer.ts

```diff
@@ -166,8 +166,8 @@ const reducer = createReducer(
   on(
     experimentsAction.actionFetchExperimentDetailStatSuccess,
     (state, { stat }) => {
-      state.stats[stat.id] = stat;
-      return state;
+      const updatedStat = state.stats[stat.id] = stat;
```

Comment:
Although this is valid, this is hard to parse as a one-liner.

Reply:
I have updated the code.